Classes/util/ColorJSONConverter.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace csharp_08
+{
+    public class ColorJsonConverter : JsonConverter
+    {
+
+        public override bool CanConvert(Type objectType)
+        {
+            return typeof(Color) == objectType;
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            // Load token from stream
+            var token = JToken.Load(reader);
+            Debug.WriteLine("token :");
+            Debug.WriteLine(token);
+
+            var target = Convert(token.Value<string>());
+
+            Debug.WriteLine("return :");
+            Debug.WriteLine(target);
+
+            return target;
+        }
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            var valueToSerialize = Convert((Color) value);
+            serializer.Serialize(writer, valueToSerialize);
+        }
+
+        private string Convert(Color value)
+        {
+            return value.IsEmpty ? string.Empty : $"#{value.R.ToString("X2")}{value.G.ToString("X2")}{value.B.ToString("X2")}";
+        }
+
+        private Color Convert(string value)
+        {
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                return Color.Empty;
+            }
+            if (value.Length != 7)
+            {
+                var message = $"Unable to convert '{value}' to a Color. Requires string length of 7 including the leading hashtag.";
```

Comment:
use string instead of var. even if we can easily understand its type, it makes more sense compared to the rest of the code